sdk/src/main/java/com/oursky/authgear/latte/LatteLink.kt

```diff
@@ -17,39 +17,55 @@ object LatteLink {
     const val BROADCAST_ACTION_LINK_RECEIVED = "com.oursky.authgear.latte.linkReceived"
     const val KEY_URI = "com.oursky.authgear.latte.linkReceived.uri"
     interface LinkHandler {
-        suspend fun handle(latte: Latte)
+        suspend fun handle(latte: Latte): LinkResult<Unit>
+    }
+
+    sealed class LinkResult<T> {
```

Comment:
There is [Result](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/) in the stdlib.

Reply:
@louischan-oursky There will be build error that tells me I should not use Result as return type, see https://stackoverflow.com/questions/52631827/why-cant-kotlin-result-be-used-as-a-return-type